BDArmory/Guidances/MissileGuidance.cs

```diff
@@ -156,7 +156,7 @@ public static Vector3 GetAirToGroundTarget(Vector3 targetPosition, Vector3 targe
             Vector3 targetAcceleration, Vessel missileVessel, float targetAlt, float maxAltitude,
             float rangeFactor, float vertVelComp, float velComp, float loftAngle, float termAngle,
             float termDist, ref int loftState, out float timeToImpact, out float targetDistance,
-            bool LoftUseAPN, float N, float minSpeed = 200)
+            MissileBase.GuidanceModes LoftGuidanceModeTerminal, float N, float minSpeed = 200)
         {
```

Comment:
Just curious, but,how does this interact with standard terminalGuidance? What happens if both are setup/enabled on the same missile?

Reply:
Shouldn't affect anything, terminalGuidance should only change the method by which the missile acquires the target, IE radar or IR, this is changing the guidance law when the missile goes terminal. The variable names for missile parameters could really use a re-organization, but given issues with compat I don't think we can do that.

Reply:
On second thought, I'll be changing this variable name and LoftTermRange as well in order to introduce a new hybrid lead/PN guidance law.